cfme/tests/services/test_rest_services.py

```diff
@@ -35,10 +35,9 @@ def services(self, request, rest_api, a_provider, dialog, service_catalogs):
         return _services(request, rest_api, a_provider, dialog, service_catalogs)
 
     @pytest.mark.uncollectif(lambda: version.current_version() < '5.7')
-    @pytest.mark.parametrize("method", ["post", "delete"])
-    def test_delete_service_dialog(self, rest_api, dialog, method):
```

Comment:
I removed test parameters because the "delete" action with DELETE method was removed from "service_dialog" collection in the last 5.7 version.